charts/buildbuddy-executor/templates/deployment.yaml

```diff
@@ -152,15 +150,7 @@ spec:
           hostPath:
             path: /var/run/docker.sock
         {{- end }}
-        - name: executor-data
```

Comment:
Looks like this volume is used above, on line 120, and in the executor config here: https://github.com/buildbuddy-io/buildbuddy-internal/blob/5ef721c4aa424e3e104be503aadef5b321586804/enterprise/config/executor.release.yaml#LL3C1-L3C1

so I don't think it's safe to remove, unless I'm misunderstanding something.

Reply:
I moved it up to line 143 to avoid the `if` blocks. It also help that we keep the order consistent with `volumeMounts` declared up top